src/commands/app/install.ts

```diff
@@ -47,11 +47,12 @@ export default class Install extends AppCLIBaseCommand {
 
       // get organization to be used
       this.sharedConfig.org =
-        this.manifestData?.organization_uid ??
-        (await getOrg(this.flags, {
-          managementSdk: this.managementSdk,
-          log: this.log,
-        }));
+        this.manifestData?.organization_uid
+          ? this.manifestData?.organization_uid !== ""
+          : await getOrg(this.flags, {
+              managementSdk: this.managementSdk,
+              log: this.log,
+            });
```

Comment:
@harshithad0703 , ternary operator seems to be used incorrectly. you can use like this. pls check similar condition in all command
const organizationUid = this.manifestData?.organization_uid;

if (!organizationUid) {
  // If `organization_uid` is null, undefined, or an empty string
  await getOrg(this.flags, {
    managementSdk: this.managementSdk,
    log: this.log,
  });
}